modules/light-clients/08-wasm/types/vm.go

```diff
@@ -40,6 +34,26 @@ func (r contractResult) Error() string {
 	return r.ErrorMsg
 }
 
+type statusQueryResponse struct {
+	contractResult
+	Status exported.Status `json:"status"`
+}
+
+type metadataQueryResponse struct {
+	contractResult
+	GenesisMetadata []clienttypes.GenesisMetadata `json:"genesis_metadata"`
+}
+
+type timestampAtHeightQueryResponse struct {
+	contractResult
+	Timestamp uint64 `json:"timestamp"`
+}
```

Comment:
In #4133 all the contract payload types are in msgs.go right? Should these be included alongside them? 

Even tho they are unexported, would be nice to have basic godocs.

Reply:
I will add them in #4133.